pkg/serviceaccount/externaljwt/plugin/testing/v1alpha1/externalsigner_mock.go

```diff
@@ -225,7 +226,7 @@ func (m *MockSigner) start(t *testing.T) error {
 
 	klog.Infof("Starting Mock Signer at socketPath %s", m.socketPath)
 	go func() {
-		if err := m.server.Serve(m.listener); err != nil {
+		if err := m.server.Serve(m.listener); err != nil && !errors.Is(err, grpc.ErrServerStopped) {
```

Comment:
> Is this fixing a race where a test starts cleaning up before `Serve()` is called? That shouldn't happen since the `start()` is followed by a wait, right?

This is used in tests that validate startup options, can fail validation when incompatible flags are used, and return an expected error and clean up the test ahead of the external server starting. That means it *is* possible for a test to correctly end and clean up concurrent with the server starting.